birdspotter/birdspotter/dataio/scripts/get_user_datasets.py

```diff
@@ -45,14 +49,28 @@ def get_dataset_data(is_authed, uuid):
                           "comments"    : [i.comments for i in shapefile_lines],
                           }
     else:
-
-        precision = 3       # aggregate data to 3 decimals points of lat/long
+        precision = 1       # aggregate data to 1 decimals points of lat/long
         precision_mod = 1  # allows for more precise tuning, >1 reduces region size <1 increases region size
         aggregation = {}
 
         for i in shapefile_lines:
             # separate by island name and by precision,
             # in case there are multiple islands in one dataset
+            if lat_min == None:
+                lat_min = i.latitude
+                lat_max = i.latitude
+                lon_min = i.longitude
+                lon_max = i.longitude
+            
+            if lat_min > i.latitude:
```

Comment:
This could probably be done cleaner using the ORM. https://stackoverflow.com/questions/10668585/how-to-get-max-value-in-django-orm/10668593